### ISSUE TITLE
Errors relating to xkbcommon

### ISSUE THREAD

**Issue by sameersharma2006** [canonical/mesa-2404#3]
I ported curtail snap to core24 & gnome-46-2404, but the snap won't start with the following error.
AFAIK from core24 onwards extensions depends on mesa-2404 stack for graphical drivers etc.
P.S. I am using Snapcraft & gnome-46-2404 from edge channel.

Any insights on this will be helpful.

```
xkbcommon: ERROR: failed to add default include path /usr/share/X11/xkb
libpxbackend-1.0.so: cannot open shared object file: No such file or directory
Failed to load module: /home/xyz/snap/curtail/common/.cache/gio-modules/libgiolibproxy.so
xkbcommon: ERROR: failed to add default include path /usr/share/X11/xkb
Segmentation fault
```

**Comment by Saviq** [comment]
Hi @sameersharma2006, you need to add `xkb-data` to your `stage-packages` - it's not part of the Mesa snap.

EDIT: and ensure you have `XKB_CONFIG_ROOT` set:

https://github.com/canonical/iot-example-graphical-snap/blob/e91466f070e817f35070029170625e4c1859aac1/snap/snapcraft.yaml#L52-L53
